test/javascripts/components/global-filter/composer-item-test.js

```diff
@@ -64,4 +64,21 @@ acceptance("Discourse Global Filter - Composer Item", function (needs) {
       "item is checked"
     );
   });
+
+  test("toggling filter items removes/adds correct tags", async function (assert) {
+    await visit("/");
+    await click("#create-topic");
+
+    // uncheck support filter
+    await click(".global-filter-composer-tag-support input");
```

Comment:
For clarity, maybe we can add an assertion here that the support tag is selected by default?